frontend/src/screens/AvailabilityCalendarUser.tsx

```diff
@@ -0,0 +1,166 @@
+    const handleEventDrop = ({ event, start, end }: { event: Event; start: Date; end: Date }) => {
+        const updatedEvents = eventsList.map((existingEvent) =>
+            existingEvent === event ? { ...existingEvent, start, end } : existingEvent
+        );
+        setEventsList(updatedEvents);
+        handleSave(updatedEvents); // Automatically save changes
+    };
+
+    const API_URL = "http://127.0.0.1:3000/";
+
+    const handleSave = async (updatedEvents: Event[]) => {
+        try {
+            // const response = await fetch(`${API_URL}/updateAvailability/${applicationId}`, {
+            //     method: "POST",
+            //     headers: { "Content-Type": "application/json" },
+            //     body: JSON.stringify({ candidate_availablity: updatedEvents }),
+            // });
+
+            // await response.json();
+        } catch (error) {
+            console.error("Error saving availability data:");
+        }
+    };
+
+    useEffect(() => {
+        const descryptId = async () => {
+            try {
+                const response = await fetch(`${API_URL}decrypt_id/${id}`);
+                const data = await response.json();
+                setApplicationId(data.data.decrypted_id);
+
+                const parsedData = data.data.candidate_availability.map((event: Event) => {
+                    // Parse the stringified JSON to get the event object
+                    const parsedEvent = JSON.parse(event as unknown as string);
+
+                    // Convert the start and end strings to Date objects
+                    return {
+                        ...parsedEvent,
+                        start: new Date(parsedEvent.start),
+                        end: new Date(parsedEvent.end),
+                    };
+                });
+
+                setEventsList(parsedData);
+            } catch (error) {
+                console.error("Error fetching availability data:", error);
+            }
+        };
+
+        descryptId();
+    }, [id]);
+
+    return (
+        // DndProvider wraps the calendar component to provide drag-and-drop functionality
+        <DndProvider backend={HTML5Backend}>
+            <div style={{ height: "80vh", padding: "20px", paddingTop: "0" }}>
+                <div
+                    style={{
+                        width: "100%",
+                        display: "flex",
+                        justifyContent: "center",
+                        marginBottom: "1rem",
+                        alignItems: "center",
+                    }}
+                >
+                    <div style={{ display: "flex", flexDirection: "column", alignItems: "center" }}>
+                        <IconButton onClick={() => navigate(-1)} sx={{ mr: 2 }}>
+                            <ArrowBackIcon />
+                        </IconButton>
+                        <h2>Enter your availability to conduct interviews</h2>
+                    </div>
+                </div>
```

Comment:
Can we change the flexDirection to be row (default) instead? Doesn't make sense to have arrow on one line and then the page title on the next

Reply:
This is what I am talking about:
![image](https://github.com/user-attachments/assets/a53a815b-3e03-4d21-9224-37322baf9a53)

Reply:
This is fixed on the other branch